scripts/build-durable-nonce.js

```diff
@@ -0,0 +1,100 @@
+const web3 = require('@solana/web3.js');
+const { binary_to_base58, base58_to_binary } = require('base58-js');
+const { exit } = require('process');
+const { createInterface } = require('readline');
+
+main();
+
+function usage() {
+  if (
+    (process.argv.length !== 3 && process.argv.length !== 4) ||
+    (process.argv[2] !== '-c' && process.argv[2] !== '-i') ||
+    (process.argv.length === 4 &&
+      process.argv[3] !== '-d' &&
+      process.argv[3] !== '-m')
+  ) {
+    console.error('Usage: ./build-nonce-account {-c|-i} [-d|-m]');
+    exit(1);
+  }
```

Comment:
process.argv[3] doesn't make sense, please refactor this to something readable instead of using raw arguments, if you need help understanding how to process command line flags, I can explain.